common/annotations/my.xml

```diff
@@ -2007,7 +2007,7 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 		<annotation cp="🚏" type="tts">ဘတ်စ်ကား မှတ်တိုင်</annotation>
 		<annotation cp="🛣">ကားလမ်း | ဟိုင်းဝေး | အမြန်လမ်း</annotation>
 		<annotation cp="🛣" type="tts">ကားလမ်း</annotation>
-		<annotation cp="🛤">ရထားသံလမြး သံလမြး ရထာျး သံလမျး | သံလမ်း</annotation>
+		<annotation cp="🛤">ရထားသံလမွး သံလမွး ရထြား သံလမြး | သံလမျး</annotation>
```

Comment:
I believe the correct version is:

ရထားသံလမ်း | ရထားလမ်း | သံလမ်း

Reply:
I was just guessing `|` is the delimiter for alternate spelling.  Please let me know if that assumption is not correct.